tests/fees.test.ts

```diff
@@ -66,7 +67,7 @@ describe("GithubCommentModule Fee Tests", () => {
 
     jest.spyOn(githubCommentModule, "_generateHtml");
 
-    const bodyContent = await githubCommentModule.getBodyContent(result);
+    const bodyContent = await githubCommentModule.getBodyContent({} as unknown as IssueActivity, result);
```

Comment:
Seems wrong but not too worried cause its in a test file.

Reply:
Yes it is just to fulfill the prototype of the function, I am only testing what's going on inside of it.